vignettes/cgdsMigration.Rmd

```diff
@@ -17,149 +18,295 @@ output:
 
 ```{r}
-searchOps(cbio, "clinical")
+getClinicalData.CGDS(x = cgds, 
+                     caseList =  "gbm_tcga_pub_expr_classical") %>%
+    datatable(options = list(pageLength = 5))
+
 ```
 
-# Genetic profiles / molecular profiles
+**Note**
+
+* We queried clinical data for `_expr_classical` cases in `gbm_tcga_pub` study id.
 
-## cgds
+## `cBioPortalData`
 
-```{r,eval=FALSE}
-getGeneticProfiles.CGDS(cgds, cancerStudy = "gbm_tcga_pub")
+* Compared to `cgdsr`, `clinicalData` function needs `studyId` and not `case_list_id` to query clinical data sets.
+
+```{r}
+## get Clinical data
+clinicalData(api = cbio, 
+             studyId = "gbm_tcga_pub") %>%
+   select(sampleId, everything()) %>%
+   datatable(options = list(pageLength = 5))
 ```
 
-## cBioPortalData
+**Note**
+
+* Explicitly we queried a subset (only `_expr_classical`, 54 samples) clinical data with `cgdsr` and all (206 samples) clinical data with `cBioPortalData`.
+
+* **`cBioPortalData` cannot query with `sampleListId` to get a subset of sample list.**
+
+* `cgdsr` returns a dataframe with `sampleId` (TCGA.02.0009.01)  but not `patientId` (TCGA.02.0009)
+
+* `cBioPortalData` returns `sampleId` (TCGA-02-0009-01) & `patientId` (TCGA-02-0009). It uses **dash** instead **dot**.
 
+
+
+# get Genetic Profiles or Molecular Profiles Identifiers {.tabset .tabset-fade .tabset-pills}
+
+## `cgdsr`
 ```{r}
-molecularProfiles(cbio, "gbm_tcga_pub")
+getGeneticProfiles.CGDS(cgds,cancerStudy = "gbm_tcga_pub" ) %>%
+    select(genetic_profile_id, genetic_profile_name, everything()) %>%
+   datatable(options = list(pageLength = 5))
 ```
 
-Note that we want to pull the `molecularProfileId` column to use in other
-queries.
+## `cBioPrtalData`
 
-# Profile data or molecular data (mRNA expression) for a set of genes
+```{r}
+molecularProfiles(api = cbio, studyId = "gbm_tcga_pub") %>%
+    select(molecularProfileId, name, everything()) %>%
+     datatable(options = list(pageLength = 5))
+```
 
-## cgds
+# Profile Data or molecular Data (mRNA expression) for a set of  genes {.tabset .tabset-fade .tabset-pills}
 
-```{r,eval=FALSE}
-getProfileData.CGDS(x = cgds, 
-    genes = c("NF1", "TP53", "ABL1"),
-    geneticProfiles = "gbm_tcga_pub_mrna", 
-    caseList = "gbm_tcga_pub_all"
-)
+## `cgdsr`
+```{r}
+#tic("cgdsr:")
+getProfileData.CGDS(x = cgds,
+                    genes = c("NF1", "TP53", "ABL1"),
+                    geneticProfiles = "gbm_tcga_pub_mrna",
+                    caseList = "gbm_tcga_pub_all") %>%
+                     datatable(options = list(pageLength = 5))
+#toc()
 ```
 
-## cBioPortalData
+## `cBioPortalData`
 
 Currently, some conversion is needed to directly use the `molecularData`
 function, if you only have Hugo symbols. First, convert to Entrez gene IDs
 and then obtain all the samples in the sample list of interest.
 
+### Convert gene HugoSymbol to Entrez
+
+It require a `cBioPortalData` version >= `2.7.11`.
```

Comment:
This is not necessary. It will be available to all Bioconductor users in the next release and current devel users.